### PR TITLE
Issue6

### LINKED ISSUE #6: Add Style
#### As a user I want to see a visually pleasing application.

__Story__:

Our web application is really starting taking shape. We have added a few
features that provide value to our customers, but some of our customers are
complaining that the website is a quite bland. It is time to spice it up by
applying a visually appealing, and easy to maintain user interface to our web
application.

__Acceptance criteria__:
- [x] The application has a header with navigation
- [x] Images are displayed in an aesthetically pleasing list of cards
- [x] Form errors are shown inline with the related input field
- [x] I know how to use Bootstrap components like buttons and navbars in my
  application.
- [x] I know the very basics of CSS and SCSS
- [x] My SCSS files are properly named and organized using BEM convention
- [ ] My app has a "sticky" footer.
- [ ] My app has a few customized styles using bootstrap variable overrides.
- [x] My app is responsive and is usable on a phone.
- [x] The active link in the navigation header is differentiated.
- [x] I use a bootstrap "container" to center my app contents (not necessarily "fluid").

__Reference material__:
- http://v4-alpha.getbootstrap.com/getting-started/introduction/
- https://github.com/plataformatec/simple_form
- http://getbem.com/naming/

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by Tianyi6679**
Closes #6

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by Tianyi6679** [comment]
[Add Style](https://trello.com/c/99FQpWlq/453-add-style)

File: app/assets/stylesheets/my_stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+html {
```
Comment (jesse-smidt) [inline app/assets/stylesheets/my_stylesheet.css:1]: my_stylesheet as a file name doesn't really carry any meaning. If these styles are global, consider placing them in `application.scss`

File: app/views/images/index.html.erb [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
   </div>
-  <% @images.each do |image| %>
+</nav>
+</header>
+<main role="main" class="container">
+  <h1 id="page-header">
+    All Images
+  </h1>
+  <%= link_to "Add your own images", new_image_path %>
+  <% @images.each_slice(2) do |imagesby2| %>
```
Comment (jesse-smidt) [inline app/views/images/index.html.erb:28]: Can you think of a way of doing this without a nested loop and just bootstrap layout?

File: app/views/images/new.html.erb [hunk 3]
```diff
@@ -1,23 +1,10 @@
-<%= form_with scope: :image, url: images_url, local: true do |form| %>
-  <p>
-    <%= form.label :title %><br>
-    <% if @image.errors.include?(:title) %>
-      <span id="title_error"><%= 'Title ' + @image.errors[:title].first %></span>
-    <% end %>
-    <%= form.text_field :title %>
-  </p>
+<%= simple_form_for @image do |f| %>
```
Comment (jesse-smidt) [inline app/views/images/new.html.erb:1]: It'd be awesome if we could have some nice bootstrap form components here! Checkout https://getbootstrap.com/docs/4.0/components/forms/

**Pull request merged**